apis/servicecatalog/generator-config.yaml

```diff
@@ -77,6 +77,10 @@ resources:
         from:
           operation: DescribeProvisionedProduct
           path: ProvisionedProductDetail.ProvisioningArtifactId
+      LastProvisioningParameters:
+        is_read_only: true
+        custom_field:
```

Comment:
Actually it's not necessary to expose all params as is, for comparison is enought to know the length of array. 
But idk how to generate custom field with type int in status.

"type: int" works only if "is_read_only" is false